drivers/net/mlx5/mlx5_vdpa.c

```diff
@@ -477,6 +483,47 @@ static int mlx5_vdpa_release_rx_steer(struct vdpa_priv *priv)
 	return 0;
 }
 
+static
+int mlx5_vdpa_create_tis(struct vdpa_priv *priv)
+{
+	uint32_t in[MLX5_ST_SZ_DW(create_tis_in)] = {0};
+	uint32_t out[MLX5_ST_SZ_DW(create_tis_out)] = {0};
+	struct mlx5_vdpa_tx_steer_info *tis = NULL;
+
+	tis = rte_zmalloc("tis", sizeof(*tis), RTE_CACHE_LINE_SIZE);
```

Comment:
I don't understand why do we need a malloced mlx5_vdpa_tx_steer_info here
You can just do the create_tis and put the devx_obj and TISN in priv->tx_steer_info right?

Reply:
right, changed